hijack/static/hijack/hijack.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
```

Comment:
```suggestion
'use strict';
```

Reply:
I usually use the [StandardJS](https://standardjs.com/) notation, which avoids semicolons where possible.